app/javascript/shared/components/__tests__/useKeyboardShortcuts.test.jsx

```diff
@@ -18,6 +18,54 @@ describe('Keyboard shortcuts for components', () => {
       expect(shortcut.KeyK).toHaveBeenCalledTimes(1);
     });
     
+    it('should fire a function when longer chained keydown is detected', async () => {
+      const shortcut = {
+        "KeyA~KeyB": jest.fn()
+      };
+
+      renderHook(() =>
+        useKeyboardShortcuts(shortcut, document, { timeout: 1500 }),
```

Comment:
Refactor (blocking): This is adding 1.5 seconds to our tests. Consider passing in a super low value, even `0`, then test with a `setTimeout` time delay of the actual timeout plus a small number, e.g. 25ms.

Reply:
The timeout inside the function isn't on an await is it still waited for?